src/app/forms/form-common-options/form-common-options.js

```diff
@@ -0,0 +1,192 @@
+export const formCommonOptions = {
+  JudicialDistrict: null,
+  petitionerRepresentativeOptions: [
+    { label: "Yes. I am the petitioner.", value: "petitioner" },
+    {
+      label: "No. I am an attorney representing the petitioner.",
+      value: "attorney"
+    }
+  ],
+  countyOptions: [
+    { label: "Beaver", value: "Beaver" },
+    { label: "Box Elder", value: "Box Elder" },
+    { label: "Cache", value: "Cache" },
+    { label: "Carbon", value: "Carbon" },
+    { label: "Daggett", value: "Daggett" },
+    { label: "Davis", value: "Davis" },
+    { label: "Duchesne", value: "Duchesne" },
+    { label: "Emery", value: "Emery" },
+    { label: "Garfield", value: "Garfield" },
+    { label: "Grand", value: "Grand" },
+    { label: "Iron", value: "Iron" },
+    { label: "Juab", value: "Juab" },
+    { label: "Kane", value: "Kane" },
+    { label: "Millard", value: "Millard" },
+    { label: "Morgan", value: "Morgan" },
+    { label: "Piute", value: "Piute" },
+    { label: "Rich", value: "Rich" },
+    { label: "Salt Lake", value: "Salt Lake" },
+    { label: "San Juan", value: "San Juan" },
+    { label: "Sanpete", value: "Sanpete" },
+    { label: "Sevier", value: "Sevier" },
+    { label: "Summit", value: "Summit" },
+    { label: "Tooele", value: "Tooele" },
+    { label: "Uintah", value: "Uintah" },
+    { label: "Utah", value: "Utah" },
+    { label: "Wasatch", value: "Wasatch" },
+    { label: "Washington", value: "Washington" },
+    { label: "Wayne", value: "Wayne" },
+    { label: "Weber", value: "Weber" }
+  ],
+  courtTypeOptions: [
+    { label: "District Court", value: "District" },
+    { label: "Justice Court", value: "Justice" }
+  ],
+  getJudicialDistrictFromCounty: function(value) {
+    switch (value) {
+      case "Box Elder":
+      case "Rich":
+      case "Cache":
+        this.JudicialDistrict = "First";
```

Comment:
Instead of setting `this.JudicialDistrict`, just return `First`. That way you don't have a temporary variable exposed on formCommonOptions that shouldn't really be used by anyone importing formCommonOptions.